### PR TITLE
Markdown versions of EMP webpages for collaborative edit

### PR DESCRIPTION
**Pull request by cuttlefishh**

### CODE REVIEW

**Comment by cuttlefishh** [comment]
@colinbrislawn here you go!

File: website/16s-rrna-amplification-protocol.md [hunk 1]
```diff
@@ -0,0 +1,201 @@
+#### Version 4\_13
+
+Primers for paired-end 16s community sequencing on the Illumina HiSeq
```
Comment (ackermag) [inline website/16s-rrna-amplification-protocol.md]: There are some that are MiSeq

File: website/16s-rrna-amplification-protocol.md [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+platform using bacteria/archaeal primers 515F/806R. Please see this
```
Comment (ackermag) [inline website/16s-rrna-amplification-protocol.md]: Some have ITS/18S

File: website/16s-rrna-amplification-protocol.md [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    Sequences](https://dl.dropboxusercontent.com/u/68839641/emp_website/515f_806rb_new.xls)
+    (515f - 806rB)
+-   [New Illumina HiSeq 16S Primer
+    Sequences](https://dl.dropboxusercontent.com/u/68839641/emp_website/515f_926r_new.xlsx)
+    (515f - 926r)
+-   [Primer Ordering and
+    Resuspension](http://press.igsb.anl.gov/earthmicrobiome/files/2013/04/EMP_primer_ordering_and_resuspension.doc)
+
+Primer Constructs designed by Greg Caporaso
+-------------------------------------------
```
Comment (ackermag) [inline website/16s-rrna-amplification-protocol.md]: These are the primers for 18S:
Linker - CA
Primer - TGATCCTTCTGCAGGTTCACCTA - barcoded
Linker - GT
Primer - GTACACACCGCCCGTC
FWD:GTACACACCGCCCGTC; REV:TGATCCTTCTGCAGGTTCACCTAC
and ITS:
ITS1/2: 
Linker (rev) AT 
Primer (reverse) GCTGCGTTCTTCATCGATGC
Linker CG
Primer (forward) CTTGGTCATTTAGAGGAAGTAA - barcoded

File: website/emp-informatics-working-group.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
+The EMP Informatics Working Group is charged with the implementation of
+data handling, analysis and publication following the generation of
+sequencing data. Their charge includes the generation and maintenance of
+the metadata repository The Global Environmental Sample Database (GESD).
+**Chief Data Analyst**: Greg Caporaso (Northern Arizona University, USA)
+**Data Analysis Team:** Sean Gibbons - Argonne National Laboratory,
+University of Chicago, USA Rob Knight - University of Colorado at
+Boulder, USA Christopher Quince - University of Glasgow, UK Alice
```
Comment (ackermag) [inline website/emp-informatics-working-group.md]: Change ROb's affiliation to UCSD

File: website/emp-informatics-working-group.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+McHardy - MPI Dusseldorf, Germany Rick Stevens, Argonne National
+Laboratory, USA **Global Environmental Sample Database Coordinator**:
+Doug Wendel - University of Colorado at Boulder, USA **GESD Team:**
+Sarah Owens - Argonne National Laboratory, USA
+<div>
+
+Gail Lesley Ackermann - University of Colorado at Boulder, USA
```
Comment (ackermag) [inline website/emp-informatics-working-group.md]: And mine to UCSD

File: website/emp-informatics-working-group.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+</div>
+
+<div>
+
+Giancarlo Galindo - Lawrence Berkley National Laboratory, USA
+
+</div>
+
+**Data Publication (MGRAST Team):** Folker Meyer - Argonne National
```
Comment (ackermag) [inline website/emp-informatics-working-group.md]: Don't believe Folker is associated any more

File: website/emp-laboratories.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
+Jack A Gilbert
+--------------
+
+Argonne National Laboratory, 9700 S Cass Avenue, Lemont, IL, 60439, USA
+gilbertjack@anl.gov
+
+Rob Knight
+----------
+
+Department of Chemistry and Biochemistry, University of Colorado at
```
Comment (ackermag) [inline website/emp-laboratories.md]: Change to : Rob Knight, Department of Pediatrics&Computer Science and Engineering, UCSD

File: website/emp-laboratories.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+Boulder, Boulder, CO, USA Rob.Knight@colorado.edu
+
+Janet Jansson
+-------------
+
+Lawrence Berkeley National Laboratory, Earth Sciences Division Berkeley,
```
Comment (ackermag) [inline website/emp-laboratories.md]: Janet is at PNNL

File: website/global-environmental-sample-database.md [hunk 9]
```diff
@@ -0,0 +1,131 @@
...
+representative of that site from that time of year. Where we do not have
+extensive temporal data, we will need to explore opportunities to obtain
+samples which expand our understanding of the temporal changes in
+community structure and their relationship to environmental conditions.
+The metadata will help us to determine where this is needed.
+
+**Submitting sample information to the GESD**
+---------------------------------------------
+
+Please contact "Gail Lesley Ackermann" \<<ackermag@ucsd.edu>\> who will
```
Comment (ackermag) [inline website/global-environmental-sample-database.md]: Should we add a sample info guide?

File: website/members.md [hunk 10]
```diff
@@ -0,0 +1,73 @@
...
+
+**Project Management and Steering Committee** comprises:
+
+**Jack A. Gilbert**, **Folker Meyer** and **Rick Stevens** from Argonne
+National Laboratory and University of Chicago, **Jonathan Eisen**
+(University of California, Davis), **Jed Fuhrman** (University of
+Southern California), **Janet Jansson** (Lawrence Berkley National
+Laboratory), **Rob Knight** and **Noah Fierer** (University of Colorado,
+Boulder), **Mark Bailey** (Center for Ecology and Hydrology, UK),
+**George Kowalchuk** (Netherlands Institute of Ecology), .
```
Comment (ackermag) [inline website/members.md]: We need to check all of these

File: website/emp-laboratories.md (hunk 7 above)
Comment (colinbrislawn) [inline website/emp-laboratories.md]: Good catch about 'Pediatrics'. I'll add that to #53

File: website/emp-laboratories.md (hunk 8 above)
Comment (colinbrislawn) [inline website/emp-laboratories.md]: I've updated Janet. Do you have Jack's info?

File: website/emp-laboratories.md (hunk 7 above)
Comment (cuttlefishh) [inline website/emp-laboratories.md]: Hi Colin, I made that change in a pull request *on your branch*. Or at least that's what I tried to do. Does that make sense?

Comment (colinbrislawn) [inline website/emp-laboratories.md]: Yep. Merged.
I'll make Gail's edits on my branch now.

File: website/16s-rrna-amplification-protocol.md (hunk 3 above)
Comment (colinbrislawn) [inline website/16s-rrna-amplification-protocol.md]: Should I put this into an excel document like the other files or should I post this directly on the page?
(You can send me an excel file, if you already have one.)

File: website/emp-informatics-working-group.md (hunk 6 above)
Comment (colinbrislawn) [inline website/emp-informatics-working-group.md]: I'm not sure...
Maybe we could send a link around and get people to check. Here is the version I'm working on:
https://github.com/colinbrislawn/emp/blob/patch-1/website/emp-informatics-working-group.md

File: website/members.md (hunk 10 above)
Comment (colinbrislawn) [inline website/members.md]: I agree. Here is the link to send around.
https://github.com/colinbrislawn/emp/blob/patch-1/website/members.md

File: website/16s-rrna-amplification-protocol.md (hunk 3 above)
Comment (cuttlefishh) [inline website/16s-rrna-amplification-protocol.md]: I think having the files hosted on the EMP site (not Dropbox) is the best. My concern with posting content on the page AND having the files is then we have the info in two places and it has to be synced.

File: website/global-environmental-sample-database.md (hunk 9 above)
Comment (colinbrislawn) [inline website/global-environmental-sample-database.md]: Yeah, that sounds helpful. I don't have this guide, though.

File: website/16s-rrna-amplification-protocol.md (hunk 3 above)
Comment (colinbrislawn) [inline website/16s-rrna-amplification-protocol.md]: I agree. Let's place all the current (new) primers in the `.md` file. For the original (old) primers, we could place them at the bottom of the page or leave them as links.

File: website/global-environmental-sample-database.md (hunk 9 above)
Comment (cuttlefishh) [inline website/global-environmental-sample-database.md]: Gail has a guide, and I have one on the EMP500 page. We just need to decide on the last few details and then merge them.

**Pull request merged**